examples/Interactive Widgets/Custom Widget - Hello World.ipynb

```diff
@@ -783,7 +785,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.4.2"
+   "version": "3.4.0"
```

Comment:
I'll just OK, on the things I have read to mark my progress on the diff view.

OK.